src/pages/basePage.ts

```diff
@@ -0,0 +1,16 @@
+import { pageFixture } from '../hooks/pageFixture';
+import AuthPage from './basicAuthPage';
+import LoginPage from './loginPage';
+
+class BasePage {
+    authPage: AuthPage;
+    loginPage: LoginPage;
+
+    public createInstances() {
+        this.authPage = new AuthPage(pageFixture.page);
+        this.loginPage = new LoginPage(pageFixture.page);
```

Comment:
I'd recommend you inject the dependency over encapsulation in this case.

Reply:
fixed